File: csharp/ButtplugCSharpFFI/ButtplugUnknownException.cs
using System;

namespace Buttplug
{
    class ButtplugUnknownException : ButtplugException
    {
        /// <inheritdoc />
        public ButtplugUnknownException()
        {
        }

        /// <inheritdoc />
        public ButtplugUnknownException(string aMessage) : base(aMessage)
        {
        }

        /// <inheritdoc />
        public ButtplugUnknownException(string aMessage, Exception aInner) : base(aMessage, aInner)
        {
        }
    }
}

File: java/lib/src/main/java/io/buttplug/ButtplugUnknownException.java
package io.buttplug;

public class ButtplugUnknownException extends ButtplugException {
    public ButtplugUnknownException(String msg) {
        super(msg);
    }

    public ButtplugUnknownException(String msg, Exception inner) {
        super(msg, inner);
    }
}
